### DISCUSSION TITLE
상점 IWP 기능으로 스크립트에 영향을 줄수 있을까요?

### DISCUSSION THREAD

**Discussion by PH62mem** [naverz/zepeto-studio-kor#927] [5. Q&A]
plushp()  <<<<이런 hp을 올려주는 함수가있는데요

상점에서 젬으로 구매를하면 plushp()가 발동되서 체력을 채워줄려고 합니다

https://studio.zepeto.me/kr/guides/multiplay-tutorial
이 가이드를 보고 거의 다 따라했는데 함수를 발동시켜주는 법은 모르겠습니다.
방법이 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 상점에서 구매를 하는 행위는 결국 버튼을 누르는 것이고
그때 iwp 구매 성공을 받으면 체력을 채워주는 직접 구현한 함수를 호출하는 구조로 구현하시면 됩니다. 
아래 가이드의 OnPurchaseComplete(item) 부분을 참고하여 주세요.
https://studio.zepeto.me/kr/guides/linking-purchasecomplete-event1
